config/dev.webpack.config.js

```diff
@@ -50,6 +50,9 @@ const webpackProxy = {
 const deps = require('../package.json').dependencies;
 const { config: webpackConfig, plugins } = config({
   rootFolder: resolve(__dirname, '../'),
+  appUrl: process.env.BETA ?  '/preview/edge' : '/edge',
+  deployment: process.env.BETA ? 'preview/apps' : 'apps',
```

Comment:
Should it be `apps` or `/apps`?

Reply:
Looks like an existing example uses `apps`.  Marking it as resolved.

Reply:
insights-dashboard has the deployment value with no leading slash https://github.com/RedHatInsights/insights-dashboard/blob/master/config/dev.webpack.config.js#L10 so I *think* its right but I don't actually know